### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by seibert**

### CODE REVIEW

File: docs/source/api.rst [hunk 1]
```diff
@@ -1,8 +1,28 @@
...
+
+   .. automethod:: intake.catalog.entry.CatalogEntry.get
 
 .. autoclass:: intake.source.base.DataSource
+   :members: close, discover, read, read_chunked, read_partition, to_dask
+
+   .. attribute:: plot
 
-.. autoclass:: intake.catalog.Catalog
+      Namespace for HoloPlot methods.  See :doc:`plotting` for more details.
```
Comment (martindurant) [inline docs/source/api.rst]: Namespace -> Accessor?

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,5 +1,13 @@
-Welcome to Intake's documentation!
-==================================
+Intake: A general interface for loading data
+============================================
+
+Intake is a lightweight set of tools for loading and sharing data in data science projects.
+Intake helps you:
+
+* Load data from a variety of formats (see :ref:`plugin-directory`) into containers you already use, like Pandas dataframes, Python lists, NumPy arrays, and more.
+* Convert your boilerplate data loading code into reusable Intake plugins
```
Comment (martindurant) [inline docs/source/index.rst]: * Load data catalogs and actual data from remote/cloud storage

Comment (martindurant) [reply docs/source/index.rst]: > Convert your boilerplate data loading code into reusable Intake plugins

You mean, a convenient place to write that boilerplate once and use many times?

File: intake/source/base.py [hunk 3]
```diff
@@ -5,13 +5,30 @@
...
         self.version = version
         self.container = container
         self.partition_access = partition_access
 
     def open(self, *args, **kwargs):
+        """Return a data source.
+        
+        Arguments are plugin specific.
+        """
         raise Exception('Implement open')
```
Comment (martindurant) [inline intake/source/base.py]: NotImplemented

File: docs/source/index.rst (hunk 2 above)
Comment (seibert) [reply docs/source/index.rst]: Yeah, going to try the word "Translate" there instead of "Convert"

**Pull request merged**